### ISSUE TITLE
the meaning of the word in BOOKREADME.md

### ISSUE THREAD

**Issue by dakeshi** [crowdsourcebooks/theswiftbook#9]
In `Who is this book for` section

> Should a term be vague, we will provide a definition of it.

I'm not a native speaker but this sentence makes me confused.
I think vague should be `clear` in the context. 
Or `To clarify a term, we will...` is more clear, I think.

**Comment by vandadnp** [comment]
I have now updated this to "Should a term be unclear, we will provide a definition of it". Thoughts?

**Comment by dakeshi** [comment]
Thank you @vandadnp 
I am sorry, I did misunderstand the meaning of sentence.
I mean, the reason I did misinterpret is the english grammar rather than words. 😭 

What I understand was :
> `Should a term be vague` == `A term must be vague`

If the sentence was not the inverted structure, I would realize it more quickly. 😞 
It's my faults @vandadnp
Thanks to your answer, I can reopen my old english grammar book. 😆
